src/nni_manager/config/config.yaml

```diff
@@ -0,0 +1 @@
+azureStorageUploadRetryCount: 1 #If upload files to azure storage failed, NNI will retry the process of uploading, this field will specify the number of attempts to re-upload files.
```

Comment:
putting it here is strange, but let's keep it for now

Reply:
changed to put it in config